schemas/rum/resource-schema.json

```diff
@@ -242,6 +242,13 @@
               "description": "trace identifier in decimal format",
               "pattern": "^[0-9]+$",
               "readOnly": true
+            },
+            "rule_psr": {
+              "type": "number",
+              "description": "tracing sample rate in decimal format",
+              "minimum": 0,
+              "exclusiveMaximum": 1,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏I think this should be inclusive?

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
              "maximum": 1,
```